docs/eigenlayer/operator-guides/operator-installation.md

```diff
@@ -241,41 +241,55 @@ Step 2: Send **at least 1 Holesky ETH** to the “address” field referenced in
 
-It will create two files: `operator.yaml` and `metadata.json` After filling the details in `metadata.json`, please upload this into a publicly accessible location and fill that url in `operator.yaml`. A valid metadata url is required for successful registration. An example operator.yaml file is provided for your reference here: [operator.yaml](https://github.com/Layr-Labs/eigenlayer-cli/blob/master/pkg/operator/config/operator-config-example.yaml) .
+The command will create two files: `operator.yaml` and `metadata.json`.
 
-A public metadata url is required to register the operator. After updating your `metadata.json` file, you must upload it to a publicly accessible location and add that public url to your `operator.yaml` file. You are also required to upload the image of the operator to a publicly accessible location and give the url in the metadata file. Operator registration only supports `.png` images for now.
+**Step 2:** Modify `metadata.json`, upload and ensure it is publicly accessible:
+
+After filling the details in `metadata.json`, please upload this into a publicly accessible location and fill that url in `operator.yaml`. Please note that a **publicly accessible** metadata url is required for successful registration. An example operator.yaml file is provided for your reference here: [operator.yaml](https://github.com/Layr-Labs/eigenlayer-cli/blob/master/pkg/operator/config/operator-config-example.yaml) .
 
 :::warning
-Please ensure that the `metadata.json`file is hosted in a publicly accessible location. The metadata_url parameter in the operator configuration must be publicly accessible in order for registration to succeed.
 When using Github for hosting this file please ensure you link to the raw file ([example](https://raw.githubusercontent.com/Layr-Labs/eigenlayer-cli/master/pkg/operator/config/metadata-example.json)), rather than the github repo URL ([example](https://github.com/Layr-Labs/eigenlayer-cli/blob/master/pkg/operator/config/metadata-example.json)).
 :::
 
+**Step 3:** Upload operator image and ensure it is publicly accessible:
+
+You are also required to upload the image of the operator to a publicly accessible location and set the url in your `metadata.json` file. Operator registration only supports `.png` images for now.
+
+**Step 4:** Configure RPC Node:  
+
 The EigenLayer CLI requires access to an Ethereum RPC node in order to post registration. Please plan to either leverage an RPC node provider or run your own local RPC node to reference in operator-config.yaml.
 
-An example list of providers is [available here](https://www.alchemy.com/list-of/rpc-node-providers-on-ethereum) and [here](https://chainlist.org/chain/17000) for your reference.
+An example list of providers is available here:
+- https://www.alchemy.com/list-of/rpc-node-providers-on-ethereum
+- https://chainlist.org/chain/17000
```

Comment:
This links only to Holesky, should it instead just link to https://chainlist.org ?

Reply:
Good call! Updated accordingly